packit/api.py

```diff
@@ -1002,6 +1002,8 @@ def sync_release(
                 self.up.local_project.git_repo.git.checkout(current_up_branch, "-f")
             self.dg.refresh_specfile()
             self.dg.local_project.git_repo.git.reset("--hard", "HEAD")
+            self.dg.local_project.git_repo.git.clean("-xdf")
```

Comment:
I love the `git clean -xfd`, but I gotta admit I've managed to wipe some files and changes I wanted to keep :smile: